corehq/apps/app_manager/xpath_validator/js/xpathValidator.js

```diff
@@ -0,0 +1,11 @@
+global.SchemeNumber = require('js-xpath/lib/schemeNumber').SchemeNumber;
```

Comment:
i think it's an antipattern to require packages from within a package. it should just be included in the project. @millerdev might have some input.

also may make sense to more this to a directory called `node` or `nodejs` instead of `js` to make it clear that this is not client side javascript